README.md

```diff
@@ -111,6 +111,24 @@ If the server list is empty in the headset app:
 - If you have a firewall, check that port 9757 (UDP and TCP) is open
 - The server and client must be compatible:
 
+## How do I use a wired connection?
+
+An issue tracking Wired WiVRn can be found [here](https://github.com/WiVRn/WiVRn/issues/30), but the current method is:
```

Comment:
As there is no hope for native libusb support, I think we can close the issue once the readme is updated, we can omit the reference to the issue in the readme.